### PR TITLE
TF-1708 Translating system mailbox

### PR DESCRIPTION
**Pull request by dab246**
### Issue

#1708 

### Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/a6ab2499-95a5-4efd-b078-1ea9fb5df745

### CODE REVIEW

File: lib/features/mailbox/presentation/extensions/presentation_mailbox_extension.dart
Comment (hoangdat) [inline lib/features/mailbox/presentation/extensions/presentation_mailbox_extension.dart:1]: ❤️

**Review by hoangdat — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

File: lib/features/mailbox/domain/state/move_mailbox_state.dart [hunk 1]
```diff
@@ -29,14 +29,17 @@ class MoveMailboxSuccess extends UIActionState {
   @override
   List<Object?> get props => [
     mailboxIdSelected,
+    moveAction,
     parentId,
-    destinationMailboxId];
+    destinationMailboxId,
+    destinationMailboxDisplayName,
+    ...super.props
```
Comment (sherlockvn) [inline lib/features/mailbox/domain/state/move_mailbox_state.dart:36]: i think it should be like this, right ?

File: lib/features/mailbox/domain/state/mark_as_mailbox_read_state.dart [hunk 2]
```diff
@@ -28,26 +28,26 @@ class UpdatingMarkAsMailboxReadState extends UIState {
...
+  MarkAsMailboxReadAllSuccess(this.mailboxDisplayName,
     {
       jmap.State? currentEmailState,
       jmap.State? currentMailboxState,
     }
   ) : super(currentMailboxState, currentEmailState);
 
   @override
-  List<Object?> get props => [mailboxName];
+  List<Object?> get props => [mailboxDisplayName];
```
[the remark this answers is not in the document]
Comment (dab246) [reply lib/features/mailbox/domain/state/mark_as_mailbox_read_state.dart:41]: good detection. It was a mistake in the previous development stage.

File: lib/features/mailbox/domain/state/move_mailbox_state.dart (hunk 1 above)
Comment (dab246) [reply lib/features/mailbox/domain/state/move_mailbox_state.dart:36]: That's right

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**